rero_ils/config.py

```diff
@@ -2805,3 +2857,42 @@ def _(x):
 
 # OAuth base template
 OAUTH2SERVER_COVER_TEMPLATE = 'rero_ils/oauth/base.html'
+
+# STOP WORDS
```

Comment:
I would specify in the comments that this section is not really about "stop word" as SEO sees it. I would rename the section: "Disregarded articles for sorting processes"